### PR TITLE
Update Contacting Cloudflare Support.md

### PR DESCRIPTION
**Pull request by AlphaK5**
sharing publicly the acceptable supported extensions

### CODE REVIEW

**Comment by AlphaK5** [comment]
PCX-7422

**Comment by cicku** [comment]
What is the size limit of the attachment?

File: content/support/Troubleshooting/General Troubleshooting/Contacting Cloudflare Support.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ card numbers, private keys, or API keys with Cloudflare.
...
+
+*Video*
+ - mp4, avi, webm
+
+*Text (only if opened with text editor)*
+
+- har, txt, csv, eml, css, html, json, tf
+
+
+Other formats than these ones, are not acceptable and the Customer Support can not proceed with the troubleshooting.
```
Comment (AlphaK5) [inline content/support/Troubleshooting/General Troubleshooting/Contacting Cloudflare Support.md:40]: Please do not upload ZIP or RAR files when sharing a HAR files, and please do not share compressed documents like DOCX, XLSX or PPTX.

The maximum file size is **20Mb.** 

If you need to share a larger file, please provide a link to the file using Google Drive or a similar sharing platform of your choice.

**Review by AlphaK5 — COMMENTED** [review]
please add the proposed text.

**Comment by Naveed9988-9** [comment]
My information submitted name Naveed iftikhar from Quetta Balochistan
Pakistan phone number 923343398851
Address. Faisal town brewery road Quetta Balochistan Pakistan
Postel coad area 87300
Bank details submitted
PK27TMFB0000000029800288
Plsss myself amount given me payment amount My given me

**Pull request closed without merging**